get-started/start-building/batch.md

```diff
@@ -36,16 +45,78 @@ Working with batch jobs in the kluster.ai API involves the following steps:
 4. **Monitor job progress** - track the status of your batch job to ensure successful completion
 5. **Retrieve results** - once the job finishes, access and process the results as needed
 
-This streamlined process enables efficient handling of large-scale requests.
-
 In addition to these core steps, this guide will give you hands-on experience to:
 
 - **Cancel a batch job** - cancel an ongoing batch job before it completes
 - **List all batch jobs** - review all of your batch jobs
 
-## Create batch jobs as JSON files
+## Quickstart snippets
+
+The following  code snippets provide a full end-to-end batch inference example for different models supported by kluster.ai. You can simply copy the snippet and paste it into your local environment. 
+
+### Python
+
+To use these snippets, run the Python script and enter your kluster.ai API key when prompted.
+
+??? example "DeepSeek R1"
+
+    ```python
+    --8<-- 'code/get-started/start-building/batch/batch-jsonl-deepseekR1.py'
+    ```
+
+??? example "LLama 3.1 8B"
+
+    ```python
+    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.1-8.py'
+    ```
+
+??? example "LLama 3.1 405B"
+
+    ```python
+    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.1-405.py'
+    ```
+
+??? example "LLama 3.3 70B"
+
+    ```python
+    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.3-70.py'
+    ```
```

Comment:
```suggestion
??? code "DeepSeek R1"

    ```python
    --8<-- 'code/get-started/start-building/batch/batch-jsonl-deepseekR1.py'
    ```

??? code "LLama 3.1 8B"

    ```python
    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.1-8.py'
    ```

??? code "LLama 3.1 405B"

    ```python
    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.1-405.py'
    ```

??? code "LLama 3.3 70B"

    ```python
    --8<-- 'code/get-started/start-building/batch/batch-jsonl-llama3.3-70.py'
    ```
```

Should use code admonitions instead of example

Reply:
@dawnkelly09 and I discussed and we prefer the graphics that is shown by default, compared to `code`. It is just more stylish for this example

Reply:
If we're going to make this a standard then we should have good reason, beyond styling, to use this over the code admonition